mailigen.gemspec

```diff
@@ -19,10 +19,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '~> 2.0'
-  
-  spec.add_dependency "activesupport"
+
+  spec.add_dependency "activesupport", ">= 3.0"
+  spec.add_dependency "retryable", ">= 2.0.4" # retry flaky remote requests
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rspec", "~> 3.5.0"
+  spec.add_development_dependency "pry", "~> 0.10.4"
```

Comment:
I think it is unnecessary dep for library. Is there any reason why it should be there?